benchmark/metrics/large_transaction_metrics_benchmark.cpp

```diff
@@ -0,0 +1,186 @@
+#include <string>
```

Comment:
It looks like the only difference between this benchmark and the original large transaction benchmark is whether the metrics are enabled. Should we put this inside the original benchmark file and pass an argument to indicate whether the metrics are enabled, to avoid duplicated code?